TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -48,6 +49,15 @@ object TeleOpMovement : Component {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // hook movement
+        if (gamepad.x) {
+            Hook.openPosition()
+
+        else:
```

Comment:
this should be an if else statement instead
```suggestion
        if else {
```